catalogue/webapp/pages/work.js

```diff
@@ -453,14 +453,13 @@ WorkPage.getInitialProps = async (context) => {
   const queryStart = asPath.indexOf('?');
   const previousQueryString = queryStart > -1 && asPath.slice(queryStart);
   const version = pageStore('toggles').apiV2 ? 2 : 1;
-  const res = await fetch(`https://api.wellcomecollection.org/catalogue/v${version}/works/${id}?includes=identifiers,items,thumbnail`);
-  const json = await res.json();
+  const work = await getWork({id, version});
 
-  if (res.status !== 200) {
-    return { statusCode: res.status };
+  if (work.type === 'Error') {
```

Comment:
I was trying to do some better union typing here i.e. `getWork(): Promise<Work | ApiError>` but hit so many snags I thought it best to get this in first.